wifiphisher/common/interfaces.py

```diff
@@ -243,6 +243,36 @@ def up_ifaces(self, ifaces):
         for i in ifaces:
             card = pyw.getcard(i.get_name())
             pyw.up(card)
+    
+    def _generate_random_address(self):
```

Comment:
I believe it makes more sense if these methods are part of the `NetworkAdapter` class instead. You should be able to do for example `ap_iface.randomize_mac`. What do you think?